control/control.rnc

```diff
@@ -294,6 +295,12 @@ precise_time =		        	element precise_time { BOOLEAN }
 ## Defines if makes sense for product to propose hibernation (jsc#SLE-12280)
 ## by default true
 propose_hibernation =	        	element propose_hibernation { BOOLEAN }
+## Defines list of default ntp servers from which product can pick.
+## Used mainly to distinguish between openSUSE and SUSE products. See bsc#1180699
+default_ntp_servers = element default_ntp_servers {
+  LIST,
+  element ntp_server { STRING }
```

Comment:
Ouch, with this declaration it only permits one server.

Reply:
And that is the reason for having this error https://github.com/yast/skelcd-control-openSUSE/pull/223/checks?check_run_id=3522806900 with the list below:

```xml
        <default_ntp_servers config:type="list">
          <ntp_server>0.opensuse.pool.ntp.org</ntp_server>
          <ntp_server>1.opensuse.pool.ntp.org</ntp_server>
          <ntp_server>2.opensuse.pool.ntp.org</ntp_server>
          <ntp_server>3.opensuse.pool.ntp.org</ntp_server>
        </default_ntp_servers>
```

Reply:
> Ouch, with this declaration it only permits one server.

So, it should be 

```
element ntp_server { STRING }+
```

Shouldn't it?

Reply:
`+` or `*` yep